proxy/proxy.go

```diff
@@ -281,11 +282,17 @@ func addCorsHeaders(header http.Header) {
 // handlePaymentRequired returns fresh challenge header fields and status code
 // to the client signaling that a payment is required to fulfil the request.
 func (p *Proxy) handlePaymentRequired(w http.ResponseWriter, r *http.Request,
-	serviceName string) {
+	serviceName string, servicePrice int64) {
 
 	addCorsHeaders(r.Header)
 
-	header, err := p.authenticator.FreshChallengeHeader(r, serviceName)
+	// Use the default price of 1 satoshi if servicePrice is zero
+	price := servicePrice
+	if servicePrice == 0 {
+		price = defaultServicePrice
```

Comment:
This would be better to set when parsing the config so that we don't have to do this every time elsewhere.

Reply:
Could you also add a debug log along the lines of `Using default LSAT price of %v per service`?